transactor.go

```diff
@@ -67,48 +66,48 @@ func (t *Transactor[B, T, O]) WithinTxWithOpts(ctx context.Context, fn func(ctx
 		nilOperator СtxOperator[T] = nil
 	)
 	if t.beginner == nilBeginner {
-		return xerrors.Errorf("transactor - can't begin: %w", ErrNilTxBeginner)
+		return fmt.Errorf("transactor - can't begin: %w", ErrNilTxBeginner)
```

Comment:
maybe `transactor - begin is nil`

because you have same error

Reply:
In this case full error msg is 
```
 transactor - can't begin: tx beginner is nil
```
and current error has the same meaning as in your note.

The second task of the wrapping `ErrNilTxBeginner` is the opportunity to check error type (using `errors.Is`)
```golfing
func Test(t *testing.T) {
	e := fmt.Errorf("transactor - can't begin: %w", ErrNilTxBeginner)
	t.Log(errors.Is(e, ErrNilTxBeginner))
}
```
```
=== RUN   Test
    transactor_test.go:564: true
--- PASS: Test (0.00s)
PASS
```
and it was designed to add opportunity of the `Transactor` users to check error and implement custom logic om the top level.